i18n/locales/es-MX/hourofcode/how-to/afterschool.md

```diff
@@ -5,121 +5,81 @@ nav: how_to_nav
 ---
 <%= view :signup_button %>
 
-# Como enseñar una Hora de Codigo en clases después de la escuela y en clubs
+# Cómo enseñar una hora de código después de la escuela con clases y en clubes
 
-Únete al movimiento e introduce a un grupo de estudiantes a su primer hora de ciencias de la computación con los siguientes pasos. La hora de código es fácil de hacer - incluso para principiantes! If you'd like an extra set of hands to help out, you can find a [local volunteer](%= codeorg_url('/volunteer/local') %) to help run an Hour of Code in your class.
+### Únete al movimiento introduciendo un grupo de estudiantes a su primer hora de ciencias de la computación con estos pasos. ¡La hora de código se ejecuta fácilmente - incluso para principiantes! If you'd like an extra set of hands to help out, you can find a [local volunteer](%= codeorg_url('/volunteer/local') %) to help run an Hour of Code in your after-school class or club.
+
+* * *
 
 ## 1. Mira este vídeo explicativo <iframe width="500" height="255" src="//www.youtube.com/embed/SrnvvWDm73k" frameborder="0" allowfullscreen mark="crwd-mark"></iframe> 
 
-## 2. Elige un tutorial
+## 2. Selecciona un tutorial
 
-Nosotros proveemos una variedad de [entretenidos tutoriales de una hora ](%= resolve_url('/learn') %) para participantes de todas las edades, creado por una variedad de socios. [!Revísalos aquí!](%= resolve_url('/learn') %)
+We provide a variety of [fun, hour-long tutorials](%= resolve_url('/learn') %) for participants all ages, created by a variety of partners. [!Revísalos aquí!](%= resolve_url('/learn') %)
```

Comment:
How come we lost these translations? It looks like the English string changed slightly like we went from `We'll host` to `We provide`. But I thought we fallback in the Hour of Code project?

Reply:
it's a property of automatic string detection in markdown in crowdin; significant enough changes in the file can actually result in the strings being identified as entirely new strings. So crowdin doesn't detect this as a string for which the english source changed, but actually as an entirely new string

Reply:
You can see the translation history for this specific string here: https://crowdin.com/project/hour-of-code/activity_stream?translation_id=169501&lang=72

Reply:
Ah, ok. Would it be ok if I go in and re-apply some of these translations that I think are still good? (I know some Spanish). Would you mind re-syncing after I do this? Or do you think not worth it?

Reply:
That seems pretty reasonable! I'd check with Suky to make sure you wouldn't be duplicating work the partners are already planning to do, but I think something like that would totally be worth it, and I'd be more than happy to re-sync

Reply:
Ok checked with Suky. I won't be duplicating any work. I just went through the es-MX Hour of Code project. All set for a re-sync!